primitiv/c/status.h

```diff
@@ -9,44 +9,14 @@
 extern "C" {
 #endif
 
-typedef enum primitiv_Code {
+typedef enum primitiv_Status {
   PRIMITIV_OK = 0,
   PRIMITIV_ERROR = 1,
```

Comment:
Could `PRIMITIV_ERROR` be a negative value (e.g.: `-1`)? It allows indicating errors as MSB (1 if error, 0 otherwise).